client/js/views/templates/AboutUs.js

```diff
@@ -1,3 +1,24 @@
 module.exports = function( { model } ) {
-    return `<div>about us</div>`
+	const images = model.reduce((memo, image) => {
+	  	const markup = `<div class='${ image.class }'>
+	    	<img data-src='${ this.ImageSrc(image.filename) }' alt='${ image.alt }'/> 
+	    	<div>${ image.date }<br/>${ image.caption }</div>
+	  	</div>`
+	  	return memo + markup
+	}, '')
+    return `<div>
+    	<div>
+	    	<div><img data-src='${ this.ImageSrc('FarmScene.jpg') }' alt='Farm Scene'/></div>
+	    	<div>
+	    		<div>Headline for About Us and the Farm</div><br/><br/>
+	    		<div>Praesent laoreet ornare ligula, ac accumsan turpis sagittis at. Integer auctor egestas eleifend. Etiam luctus mattis justo, vitae fermentum libero euismod lacinia. Proin at consequat risus. Praesent sollicitudin vestibulum felis, ut sodales enim.</div><br/>
```

Comment:
For paragraphs, use `<p>` elements, no `<br>` necessary again because a `<p>` is a block. It will have a certain amount of `margin-bottom` built in by the browser, which can be adjusted if necessary.